Tests/MusicXMLTests/Complex Types/DirectionTests.swift

```diff
@@ -46,4 +46,55 @@ class DirectionTests: XCTestCase {
         )
         XCTAssertEqual(decoded, expected)
     }
+
+    func decodingDirection_wedge() throws {
```

Comment:
I think these need to be named `testDecodingDirection_wedge()` or similar in order for `XCTest` to run them?

Reply:
oh damn